packages/sdks/javascript/package.json

```diff
@@ -21,7 +21,10 @@
     "prepublishOnly": "npm test",
     "prepare": "npm run build",
     "preversion": "npm run lint",
-    "test": "jest"
+    "test": "jest",
+    "docker:build": "docker build --force-rm -f Dockerfile-dev -t devopness_sdk-javascript .",
+    "docker:test": "docker run devopness_sdk-javascript npm run test",
+    "docker:build-api-models": "docker run -v sdk-javascript:/usr/app/ devopness_sdk-javascript npm run build-api-models"
```

Comment:
Why adding those scripts here if a `Makefile` has already been added to the project?
Devs should be encouraged to simply run `make` or `make build`, otherwise they will still need to have `node/npm` on their host machines, reducing the advantage of using Docker containers here.